tchannel-core/src/main/java/com/uber/tchannel/handlers/RequestRouter.java

```diff
@@ -119,6 +119,10 @@ protected void messageReceived(final ChannelHandlerContext ctx, final Request re
         // Get handler for this method
         RequestHandler handler = this.getRequestHandler(service, endpoint);
         if (handler == null) {
+            handler = topChannel.getDefaultHandler();
```

Comment:
why default handler can be null? You need to handle the req regardless, otherwise, there will be buffer leak